docs/includes/apiargs-MongoDBCollection-method-watch-option.yaml

```diff
@@ -43,6 +43,10 @@ interface: phpmethod
 operation: ~
 optional: true
 ---
+source:
+  file: apiargs-MongoDBCollection-common-option.yaml
+  ref: readConcern
+---
```

Comment:
It looks like this was never added in https://github.com/mongodb/mongo-php-library/pull/439 and I missed it in the relevant PR.